### PR TITLE
Fix and Update the pending breakpoint functionality.

### PR DESCRIPTION
**Pull request by knightburton**
IoT.jsCode-DCO-1.0-Signed-off-by: Imre Kiss kissi.szeged@partner.samsung.com

### CODE REVIEW

File: src/app/client-parsesource.js [hunk 1]
```diff
@@ -203,6 +203,35 @@ export default class ParseSource {
...
+            this._debuggerObj.deletePendingBreakpoint(index);
+          }
+        }
+      });
+    } else {
+      this._debuggerObj.sendParserConfig(0);
+    }
+
     this._alive = false;
+    return;
```
Comment (rtakacs) [inline src/app/client-parsesource.js]: I think this `return` is not necessary.

**Comment by knightburton** [comment]
@rtakacs I've updated the PR based on your comment.

**Comment by rtakacs** [comment]
LGTM

**Comment by yichoi** [comment]
LGTM

**Pull request merged**